### PR TITLE
Respect max_alarm_data_length

### PR DESCRIPTION
**Pull request by pyhys**
Also use a define for the max_alarm_data_length we send back to the PLC

### CODE REVIEW

**Review by olbjo — APPROVED** [review]
👍 
I think the todo inpf__alarm_send_alarm() shall be completed before merge.

**Comment by pyhys** [comment]
Yes, I should probably wait for that branch to be merged.

**Comment by pyhys** [comment]
Now updated `pf_alarm_send_alarm() `

**Pull request merged**
